src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -470,17 +470,33 @@ private DocumentDatabase ActuallyCreateDatabase(StringSegment databaseName, Rave
             }
         }
 
+        public ConcurrentDictionary<string, ConcurrentQueue<string>> InitLog = new ConcurrentDictionary<string, ConcurrentQueue<string>>();
+
         private DocumentDatabase CreateDocumentsStorage(StringSegment databaseName, RavenConfiguration config)
         {
             try
             {
+                if (InitLog.TryGetValue(databaseName, out var initLogQueue) == false)
+                {
+                    initLogQueue = new ConcurrentQueue<string>();
+                    InitLog[databaseName] = initLogQueue;
+                }
+                else
+                {
+                    initLogQueue.Clear();
```

Comment:
This shouldn't be possible. You should have just one instance here per db, only while it is loading